engine/src/main/java/org/openobservatory/engine/PESession.java

```diff
@@ -3,7 +3,7 @@
 import oonimkall.Oonimkall;
 import oonimkall.Session;
 
-final class PESession implements OONISession {
+public final class PESession implements OONISession {
```

Comment:
Think this was a trial and error attempt at a solution, no need to keep it

Reply:
Then revert it ;)

Reply:
waiting for other feedbacks